packages/cli/src/index.ts

```diff
@@ -130,6 +134,23 @@ ECOM_STORE_ID=${storeId}
       }
     }
 
+    let createAllSecrets = false;
+    if (GITHUB_TOKEN) {
```

Comment:
```suggestion
    if (GITHUB_TOKEN && argv.github !== false) {
```

A exemplo do https://github.com/ecomplus/cloud-commerce/blob/ffa13dbcf2c95b24e26f680e61ecdb3e0e299b9f/packages/cli/src/index.ts#L128

`--no-github` (ou talvez um nome melhor pra opção 🙃 ) skipa esse passo